_posts/2016-10-31-Kickstarters-Project.md

```diff
@@ -0,0 +1,205 @@
+---
+layout: post
+authors: [tim_verte]
+title: 'Kickstarter project 2016'
+image:
+tags: [Spring, Angular2, Unit Test, Postman, Spring Boot, Spring REST Docs]
+category: Domain-Driven-Design
+comments: true
+---
+
+>On August 1<sup>'st</sup> it was D-day for all the kickstarters that had recently joined Ordina. A batch of talented new people were ready to embark on a new adventure. This year around 50 people joined Ordina and participated in the Kickstarter Project. The JWorks kickstarter group consisted of 7 people, all of which were eager to get started. 6 people joined the JWorks unit and 1 joined the Security unit. The purpose of the two month long kickstarter protject is to broaden the knowledge of and prepare the kickstarters for their first project.
+
+# Kickstarter project 2016
+
+## First impressions
+
+<q>You never get a second chance to make a first impression.</q>
+<br/>
+-- <cite>Harlan Hogan</cite>
+
+And boy Ordina did a pretty good job!
+
+The reception on the first day was really great and pretty informal.
+First off the kickstarters received a tour of the company.
+They introduced themselves and got to know eachother in a pretty playful way.
+FINALLY the moment had arrived that everybody was waiting for !
+The kickstarters received their highly anticipated company car and laptop.
+
+In the office the overall atmosphere is pretty loose, you can ask anyone anything and you can talk to everybody.
+
+After a few days the kickstarters also got the chance to go on a teambuilding day in Mechelen.
+During the course of this day they had to work together as a team to complete some questions and games.
+The winners were rewarded with a cup. Which ofcourse had a positive effect on learning how to communicate in a team and under stress, because some of the tests had to be completed within an certain amount of time.
+
+Most of the kickstarters already had the chance to go to one of the interesting events organised by Ordina, where there are lots of interesting people to network with.
+And a lot of great food ofcourse.
+
+
+
+## August - Learning
+
+The focus during the first month of the Kickstarters project was primarily on learning.
```

Comment:
"The first month of the Kickstarters project the focus was primarily on learning."

Otherwise it looks like you're learning about "the first month of the Kickstarter project"